wireless_watcher/launch/watcher.launch

```diff
@@ -1,3 +1,4 @@
 <launch>
-  <node ns="wireless" pkg="wireless_watcher" name="wireless_watcher" type="watcher_node" />
+  <arg name="robot_name" default="$(optenv ROBOT p3_001)"/>
```

Comment:
you don't need to read ROBOT env var in yet another place, take it as an arg from a top level launch file.